compiler/rustc_codegen_llvm/src/va_arg.rs

```diff
@@ -410,6 +581,7 @@ pub(super) fn emit_va_arg<'ll, 'tcx>(
             let indirect: bool = target_ty_size > 8 || !target_ty_size.is_power_of_two();
             emit_ptr_va_arg(bx, addr, target_ty, indirect, Align::from_bytes(8).unwrap(), false)
         }
+        "x86_64" if !target.is_like_darwin => emit_x86_64_sysv64_va_arg(bx, addr, target_ty),
```

Comment:
to support `extended_varargs_abi_support` in `c_variadics`, I think we'll have to split this match up a la "strategy pattern", but I guess that can be another PR.